src/main/java/seedu/ui/UiCommand.java

```diff
@@ -72,11 +72,11 @@ public void addUpdatedField(String updatedField, String updatedValue) {
     public void addInvalidFlag(String flag) {
         String newInvalidFlags = getInvalidFlags();
         switch (flag) {
-        case "name":
-            newInvalidFlags += "Role not specified." + "\n";
+        case "role":
+            newInvalidFlags += "Role not specified. Internship not added." + "\n";
```

Comment:
Perhaps have a separate function for printing the exact message when a addCommand fails, this function is meant for other commands too